src/main/java/de/fraunhofer/aisec/codyze/config/Configuration.kt

```diff
@@ -228,7 +235,16 @@ class Configuration {
         @JvmStatic
         fun initConfig(configFile: File?, vararg args: String?): Configuration {
             val config: Configuration =
-                if (configFile != null) parseFile(configFile) else Configuration()
+                if (configFile != null) {
+                    parseFile(configFile)
+                } else {
+                    val defaultConfigFile = File("config.yaml")
```

Comment:
Rename to `codyze.yaml` and better use common property to make it consistent with `Main.ConfigFilePath.configFile`.